components/purchase_modal/index.ts

```diff
@@ -48,6 +48,8 @@ function mapStateToProps(state: GlobalState) {
         currentTeam: getCurrentTeam(state),
         theme: getTheme(state),
         isDelinquencyModal,
+        annualSubscription: state.entities.general.config.FeatureFlagAnnualSubscription === 'true',
+        usersCount: Number(state.entities.admin.analytics!.TOTAL_USERS) || 1,
```

Comment:
In general, it is preferred to use the relevant selector and create one if none exists. For usersCount, we can use [getConfig](https://github.com/mattermost/mattermost-webapp/blob/master/packages/mattermost-redux/src/selectors/entities/admin.ts#L16), andn for annualSubscription, either [getConfig](https://github.com/mattermost/mattermost-webapp/blob/master/packages/mattermost-redux/src/selectors/entities/general.ts#L13) or [getFeatureFlagValue](https://github.com/mattermost/mattermost-webapp/blob/master/packages/mattermost-redux/src/selectors/entities/general.ts#L20)

Reply:
For the user count, isn't getConfig getting `state.entities.general.config` and the current user count that I'm using is `state.entities.admin.analytics!.TOTAL_USERS`. Is there another variable in `state.entities.general.config` that I should be using instead? Or should I be making a selector for `state.entities.admin.analytics`?

Reply:
There are two getConfig selectors, and I linked both, which is confusing. The first one will get the admin config with the user count. The second one can be used to get the feature flag value (or you could use getFeatureFlagValue instead to do that). If you want both getConfig's, then for importing, you can rename one of them, i.e. you could do `import {getConfig as getAdminConfig} from 'mattermost-redux/selectors/entities/admin'`

Reply:
Oh I didn't realize the 2 configs were different. Thanks